src/component/layout/BackButton/StyledBackButton.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import styled from "styled-components";
+import BackButtonSvg from "../../../images/arrow-back.svg";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+
+const BackButtonWrapper = styled.div`
```

Comment:
why do you style elements here? I used to use .css files for styling

Reply:
It's very good question. In this case, the CSS styles was not moved to a separate file because it's a reusable **styled component**. You can learn more about reusable **styled components** [here](https://styled-components.com/).

Reply:
Have you also thought about using https://tailwindcss.com/, https://getbootstrap.com/ or https://mui.com/? Could save y'all time. They have pre-made components like navbar and form and also include inline styling at the class level.